cfme/tests/cloud/test_provisioning.py

```diff
@@ -207,6 +207,107 @@ def _finished():
         num_sec=600, delay=5, message="VM {} becomes visible".format(vm_name))
 
 
+@pytest.mark.uncollectif(lambda provider: provider.type != 'ec2' or current_version() < "5.7")
+def test_ec2_manual_placement_using_rest(
+        request, setup_provider, provider, vm_name, rest_api, provisioning):
+    """ Tests provisioning ec2 instance with manual placement using the REST API.
+
+    Metadata:
+        test_flag: provision
+    """
+    image_guid = rest_api.collections.templates.get(name=provisioning['image']['name']).guid
+    provider_id = rest_api.collections.providers.get(name=provider.name).id
+    instance_type = provisioning['instance_type'].split(':')[0].strip()
+
+    flavors = rest_api.collections.flavors.find_by(name=instance_type)
+    assert len(flavors) > 0
+    flavor = None
```

Comment:
I'm using the loop variable later in the code. This assignment is there to make it explicit that the loop variable is defined even in case when there's nothing to iterate over. It's not necessary at the moment as there's the assert above and else in the for loop, but pylint still insists. So at the moment it's there just to make pylint happy (otherwise it reports ``[undefined-loop-variable] Using possibly undefined loop variable 'flavor'``).